README.md

```diff
@@ -129,6 +129,11 @@ Click the button below to start deployment.
    **Note:** This value defaults to `insight-operational-logs`. This Event Hub is created automatically by Azure when a subscription [Log Profile is integrated with Event Hub through the Azure Monitor service](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/stream-monitoring-data-event-hubs#azure-subscription-monitoring-data).
    Follow this guide to [Stream the Azure Activity Log to Event Hubs](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/activity-logs-stream-event-hubs).
 
+   - **Event Hub Retention** - Type the number of data retention days between 1 and 7. The value applies only if new event hub is created.
```

Comment:
Is this safe? We're seeing delays of up to 2 days and we still have very limited production data (that is, we don't know how the collector will behave in different customer environments). The default is 7, but maybe this option isn't worth surfacing until customers don't ask for it?

Reply:
I think we need a note here similar to that in L122; something like:

 **Note:** The following template parameters are optional and need to be entered only if creating a new Event Hub:

Reply:
OR:
The following template parameters are mandatory if creating a new Event Hub: